demos/grouped_mask_prostate_longitudinal/demo_predict.py

```diff
@@ -48,8 +49,8 @@
     config_path.append("config/test/demo_unpaired_grouped.yaml")
 
 predict(
-    gpu="0",
-    gpu_allow_growth=True,
+    gpu="",
```

Comment:
why changing this, it should no be linked to this PR?

Reply:
yerh…it should be my mistake.